docs/_source/practical_guides/create_dataset.md

```diff
@@ -52,7 +52,61 @@ fields = [
 The order of the fields in the UI follows the order in which these are added to the `fields` attribute in the Python SDK.
 ```
 
-##### Define `questions`
+#### Define metadata properties
+
+Metadata properties allow you to configure the use of metadata information for the filtering and sorting features available in the UI and Python SDK.
+
+You can define metadata properties using the Python SDK by providing the following arguments:
+
+- `name`: The name of the metadata property, as it will be used internally.
+- `title` (optional): The name of the metadata property, as it will be displayed in the UI. Defaults to the `name` value, but capitalized.
+- `visible_for_annotators` (optional): A boolean to specify whether the metadata property will be accessible for users with an `annotator` role in the UI (`True`), or if it will only be visible for users with `owner` or `admin` roles (`False`). It is set to `True` by default.
+
+The following arguments apply to specific metadata types:
+- `values` (optional): In a `TermsMetadataProperty`, you can pass a list of valid values for this metadata property, in case you want to run a validation. If none are provided, the list of values will be computed from the values provided in the records.
+- `min` (optional): In an `IntengerMetadataProperty` or a `FloatMetadataProperty`, you can pass a minimum valid value. If none is provided, the minimum value will be computed from the values provided in the records.
+- `max` (optional): In an `IntengerMetadataProperty` or a `FloatMetadataProperty`, you can pass a maximum valid value. If none is provided, the maximum value will be computed from the values provided in the records.
+
+::::{tab-set}
```

Comment:
normally we keep this in `_common/tabs` to allow for re-usability of such snippets